.github/pull_request_template.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Code smission
+about: Add code or modify existing code in CliMa
+---
+
+<!--
+Thanks for submitting code to CliMa, the Climate Machine.
+
+Before continuing, please be sure you have read the [How to contribute guide for code submission](We should have this) and have:
+
+1. Written and run all necessary tests with CLIMA by including `tests/runtests.jl`
+2. Followed all necessary [style guidelines](We should have this)
+3. Identified key contributors to review this submission
+
+If you would like to contact us, we are also available on Slack
+-->
+
+# Code submission
+
+## Description
+
+A clear and concise description of the code with usage.
+
+<!--- Please leave the following section --->
+
+## For CliMA Developers
```

Comment:
Does this mean "for code review"?

Reply:
Yeah, it's just supposed to be for review. I'll change it to `CliMA developer review`?